src/main/java/Command.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Execute the command in Duke.
+ */
+public class Command {
```

Comment:
The command class could be an abstract class. Methods such as execute do not need to be fully described and left for the child classes to define since each of them exhibit different behaviour.